src/lib/draw-and-pick.js

```diff
@@ -196,6 +196,15 @@ export function pickLayers(gl, {
     if (info) {
       infos.set(info.layer.id, info);
     }
+
+    // update pickingSelectedColor so picked model is highlighted.
+    if (layer.props.pickingHighlightColor !== NULL_PICKING_COLOR) {
```

Comment:
The layer class sets every prop to a uniform (updateModuleSettings). So for `null` to work, we can't overload single prop to specify null and a vector, lets discuss this offline.